### PR TITLE
message validation: add comments when getting first validator index

### PR DESCRIPTION
**Pull request by nkryuchkov**
Improve clarity of the check by adding a comment

### CODE REVIEW

**Comment by nkryuchkov** [comment]
@MatheusFranco99 Should we add `ErrProposalSCCSeveralIndices` to the knowledge base?

File: message/validation/common_checks.go [hunk 1]
```diff
@@ -98,18 +98,24 @@ func (mv *messageValidator) validateBeaconDuty(
 	slot phase0.Slot,
 	indices []phase0.ValidatorIndex,
 ) error {
+	if len(indices) != 1 {
+		return ErrProposalSCCSeveralIndices
+	}
+
+	index := indices[0]
```
Comment (y0sher) [inline message/validation/common_checks.go:105]: from what I've seen in the code, we are getting the indicies from the committeeInfo struct which is created by getting all the committee indices from the validatorStore, I don't userstand how can you be sure that the first index is the correct validator that has the proposal duty?

**Comment by y0sher** [comment]
not needed.

**Comment by nkryuchkov** [comment]
@y0sher Let's add a comment instead

**Review by y0sher — APPROVED** [review]

**Pull request merged**